hdf5-vol-daos.spec

```diff
@@ -0,0 +1,174 @@
+%global with_mpich 1
+%global with_openmpi3 1
+
+%if %{with_mpich}
+%global mpi_list mpich
+%endif
+%if %{with_openmpi3}
+%global mpi_list %{?mpi_list} openmpi3
+%endif
+
+%if (0%{?suse_version} >= 1500)
+%global module_load() if [ "%{1}" == "openmpi3" ]; then module load gnu-openmpi; else module load gnu-%{1}; fi
+%else
+%global module_load() module load mpi/%{1}-%{_arch}
+%endif
+
+Name:    hdf5-vol-daos
+Version: 1.0
+Release: 1.gb324b90d455%{?dist}
+Summary: A Multi-purpose, Application-Centric, Scalable I/O Proxy Application
+
+License: GPL
+URL:     https://github.com/daos-stack/hdf5-vol-daos
+Source0: daos-vol-master@b324b90d455.zip
+Source1: vol-tests-master@31660ab1935.zip
```

Comment:
These need to be URLs to where the sources can be downloaded.  Also, `.tar.*` is preferred over ZIP archives due to the potential incompatible implementations of ZIP.

Reply:
Waiting for HDF5 to push code to github